### ISSUE TITLE
Snap package bugs

### ISSUE THREAD

**Issue by qarmin** [qarmin/czkawka#217]
- [ ] - External drives are not available
- [ ] - Some folders like `/home/user/.kodi` are not available

Also Gnome-3-34 SDK is broken so I am forced to use Gnome-3-28 SDK.

I will try to drop use of Gnome extension.

**Comment by alexislefebvre** [comment]
Does it work after allowing access to `removable-media`?

> sudo snap connect <app>:removable-media :removable-media

Source: https://askubuntu.com/questions/896511/is-there-a-way-for-a-snap-application-to-access-a-network-mounted-disk-drive/896973#896973

**Comment by qarmin** [comment]
Looks that after executing this command I'm able to open external drives. 
This should work out of box, because I added this plug to `snapcraft.yaml` build file.

I don't have a time to play in this snap game - "fixing bugs which shouldn't happen, but happens", so in this commit https://github.com/qarmin/czkawka/commit/6bdd0cce7c278474c7e7b27b9a05ee885aa82bbd I removed Snap sanbox.

Let's see if this will help with most problems.
